src/sundials/fmod_int64/fsundials_core_mod.f90

```diff
@@ -47,6 +47,8 @@ module fsundials_core_mod
  integer(C_INT), parameter, public :: SUN_COMM_NULL = 0_C_INT
 #endif
 
+ type(C_PTR), parameter, public :: SUN_REAL_FORMAT_E = "%.DBL_DIGe"_C_PTR
```

Comment:
Looks like swig is having trouble converting this macro. Is there a way to exclude this?